CHANGELOG.md

```diff
@@ -19,6 +19,250 @@
 
 # OpenWhisk Core
 
+## Apache 2.0.0
+### Branch: [2.0.0](https://github.com/apache/openwhisk/tree/2.0.0)
+### Changes
```

Comment:
Since there are too many changes, I couldn't pick notable changes but list all of them with the automation tool.